packages/clay-select/src/SelectWithOption.tsx

```diff
@@ -18,7 +18,7 @@ interface IProps extends React.ComponentProps<typeof Select> {
 const SelectWithOption: React.FunctionComponent<IProps> = ({
 	options = [],
 	...otherProps
-}) => (
+}: IProps) => (
```

Comment:
Shouldn't need this since we have line 18.

Reply:
It's used for react-docgen